client/components/marketing-survey/cancel-purchase-form/step-components/test/upgrade-at-step.jsx

```diff
@@ -1,67 +1,57 @@
-					recordTracksEvent={ noop }
-					translate={ translate }
-					selectedSite={ selectedSite }
-				/>
-			);
-		} );
-
-		test( 'should render translated heading content', () => {
-			expect( wrapper.find( 'FormSectionHeading' ).props().children ).toEqual(
-				'Translated: New! Install Custom Plugins and Themes'
-			);
-		} );
+	test( 'should render translated heading content', () => {
+		const { container } = render( <UpgradeATStep { ...props } recordTracksEvent={ noop } /> );
+		expect( container.firstChild ).toHaveTextContent(
+			'Translated: New! Install Custom Plugins and Themes'
+		);
+	} );
 
-		test( 'should render translated link content', () => {
-			expect( wrapper.find( 'FormFieldset > p' ).props().children ).toEqual(
-				'Translated: Did you know that you can now use third-party plugins and themes on the WordPress.com Business plan? ' +
-					'Claim a 25% discount when you upgrade your site today - {{b}}enter the code BIZC25 at checkout{{/b}}.'
-			);
-		} );
+	test( 'should render translated link content', () => {
+		render( <UpgradeATStep { ...props } recordTracksEvent={ noop } /> );
+		expect( screen.queryByRole( 'group' ) ).toHaveTextContent(
+			'Translated: Did you know that you can now use third-party plugins and themes on the WordPress.com Business plan? ' +
+				'Claim a 25% discount when you upgrade your site today - {{b}}enter the code BIZC25 at checkout{{/b}}.'
+		);
+	} );
 
-		test( 'should render translated confirmation content', () => {
-			expect( wrapper.find( 'FormFieldset > ForwardRef(Button)' ).props().children ).toEqual(
-				'Translated: Upgrade My Site'
-			);
-		} );
+	test( 'should render translated confirmation content', () => {
+		render( <UpgradeATStep { ...props } recordTracksEvent={ noop } /> );
+		expect( screen.queryByRole( 'link' ) ).toHaveTextContent( 'Translated: Upgrade My Site' );
 	} );
 
 	test( 'should render button with link to business plan checkout', () => {
-		const wrapper = shallow(
-			<UpgradeATStep recordTracksEvent={ noop } translate={ noop } selectedSite={ selectedSite } />
-		);
+		render( <UpgradeATStep { ...props } recordTracksEvent={ noop } translate={ noop } /> );
 
-		expect( wrapper.find( Button ).props().href ).toEqual(
+		expect( screen.queryByRole( 'link' ) ).toHaveAttribute(
+			'href',
 			'/checkout/site_slug/business?coupon=BIZC25'
 		);
 	} );
 
-	test( 'should fire tracks event when button is clicked', () => {
+	test( 'should fire tracks event when button is clicked', async () => {
 		const recordTracksEvent = jest.fn();
-		const wrapper = shallow(
-			<UpgradeATStep
-				recordTracksEvent={ recordTracksEvent }
-				translate={ noop }
-				selectedSite={ selectedSite }
-			/>
+		render(
+			<UpgradeATStep { ...props } translate={ noop } recordTracksEvent={ recordTracksEvent } />
 		);
+		const btn = screen.queryByRole( 'link' );
 
-		wrapper.find( Button ).simulate( 'click' );
+		btn.addEventListener( 'click', ( event ) => event.preventDefault(), false );
```

Comment:
I ran into `Error: Not implemented: navigation (except hash changes)` when trying to simulate a click on the anchor. I implemented the above solution found here: https://github.com/jsdom/jsdom/issues/2112#issuecomment-926601210